services/tests/root.test.ts

```diff
@@ -0,0 +1,21 @@
+import MockAdapter from "axios-mock-adapter";
+import { getApiClient } from "../../utils/apiUtils";
+import { getRecommendations } from "../root";
+
+describe("root tests", () => {
+  it("should ensure it returns data when response is ok", async () => {
+    const mock = new MockAdapter(getApiClient().axiosInstance);
+    const resData = { results: [{ trackId: 12345, trackName: "Baarishein" }] };
+    mock.onGet(`/search?term=anuv`).reply(200, resData);
+    const res = await getRecommendations();
+    expect(res).toEqual(resData.results);
+  });
+
+  it("should ensure it returns an empty array when err coming in response ", async () => {
```

Comment:
```suggestion
  it("should return an empty array if error is present in the response", async () => {
```

Reply:
Thanks for the review. Updated accordingly.